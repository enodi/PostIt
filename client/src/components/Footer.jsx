import React from 'react';
import { Link } from 'react-router';


const Footer = () => (
  <div>
  <br/>
    <footer className="page-footer #000000 black">
      <div className="container">
        <div className="row">
          <div className="col l4 s12">
            <h5><Link to="/" className="white-text">PostIt</Link></h5>
            <ul>
              <li className="grey-text text-lighten-4">&copy; 2017</li>
            </ul>
          </div>
        </div>
      </div>
      <div className="footer-copyright"></div>
    </footer>
  </div>
);

export default Footer;
