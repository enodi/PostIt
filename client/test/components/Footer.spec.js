import React from 'react';
import { shallow } from 'enzyme';

import Footer from '../../src/components/Footer.jsx';

describe('Footer Component', () => {
  it('should render Footer Component', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
