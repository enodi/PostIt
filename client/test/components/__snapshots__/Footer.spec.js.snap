// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render Footer Component 1`] = `
<div>
  <br />
  <footer
    className="page-footer #000000 black"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col l4 s12"
        >
          <h5>
            <Link
              className="white-text"
              onlyActiveOnIndex={false}
              style={Object {}}
              to="/"
            >
              PostIt
            </Link>
          </h5>
          <ul>
            <li
              className="grey-text text-lighten-4"
            >
              Â© 2017
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="footer-copyright"
    />
  </footer>
</div>
`;
